migtests/tests/pg-tests/pg-codependent-schemas/validate

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+
+import yb
+
+def main():
+	yb.run_checks(migration_completed_checks)
+
+
+#=============================================================================
+
+EXPECTED_ROW_COUNT_PUBLIC = {
+	'audit2': 4,              
+	'foreign_test': 2,
+ 	'list_active': 2,  
+ 	'list_archived': 1,  
+ 	'list_part': 5,   
+	'primary_test': 3,
+	'tt':4,
+	'v1':2,
+	'mv1':2,
+}
+EXPECTED_ROW_COUNT_SCHEMA2 = {
+	'audit': 4,
+	'foreign_test': 3,
+	'list_others': 2, 
+	'primary_test': 2,
+	'v1': 2,
+}
+
+def check_validations_per_schema(tgt,schema_name):
+
+	EXPECTED_ROW_COUNT = EXPECTED_ROW_COUNT_PUBLIC if schema_name == 'public' else EXPECTED_ROW_COUNT_SCHEMA2
+
+	table_list = tgt.get_table_names(schema_name)
+	print("table_list:", table_list)
+	assert (len(table_list) == 7 and schema_name == 'public') or (len(table_list) == 4 and schema_name == 'schema2')
```

Comment:
I see many such conditions. This defeats the purpose of creating a general function. Arrange the expected values in a dictionary keyed by the schema name:
```
EXPECTED = {
  "public": {
    "table_count": 7,
    "mview_count": 1,
  },
  "schema2": {
    "table_count": 4,
    "mview_count": 0,
  }
}
```
Then you can easily write a condition as:
```
assert len(table_list) == EXPECTED[schema_name]["table_count"]
assert len(mview_list) == EXPECTED[schema_name]["mview_count"]
```